app/templates/merchant/view_listings.html

```diff
@@ -50,11 +81,14 @@
         </div>
         <br>
         <button class="ui button" type="submit">Search</button>
+        <a class="ui button violet" href="{{ url_for('merchant.listing_view_all', page=1) }}">Clear Criteria</a>
```

Comment:
lol, this is small, but I think it makes more sense for the search button to be vibrant, and the clear button to be subdued. Maybe make the search button blue and the clear button grey. Thoughts?